openscap/src/main/resources/antisamy.xml

```diff
@@ -85,7 +85,7 @@ http://www.w3.org/TR/html401/struct/global.html
 		<regexp name="cssPseudoElementSelector" value=":[a-zA-Z0-9\-_\(\)\:]+"/>
 
 		<!--  This is to list out any psuedo-element names that are *not* valid -->
-		<regexp name="cssPseudoElementExclusion" value=""/>
+		<regexp name="cssPsuedoElementExclusion" value=""/>
```

Comment:
here is a typo on the name did you wrote the file @ncharles or it came with the dependency ?

Reply:
it cale with the dependency :(